### DISCUSSION TITLE
In Greek mythology, who is ⚡

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#96] [Galaxy Qs] [answered]
In Greek mythology, who is the king of the gods ?

**Comment by rezzcode** [comment, marked as answer]
Its Zeus the ruler of mount Olympus

**Reply by Sriansh-raj** [reply]
Congrats @the-1Riddle for earning the `Galaxy Brain🥉` achievement 🎉

**Reply by Sriansh-raj** [reply]
Don't forget to create a PR and update your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to create a history 😃
